src/session-summaries/the-intersection-observer-api.md

```diff
@@ -0,0 +1,12 @@
+## Intersection Observer Api
+
+This Video introduces the basics of the **Intersection Observer API** and what it can be used for while taking the user through real world examples which were explained carefully. The Intersection Observer API simply watches an element based on some options passed to it during instantiation and calls the callback function also passed to it on instantiation when certain conditions are met.
```

Comment:
It's best to avoid words like simple, easy etc in technical documents. Those are highly relative